ethernet_handler.py

```diff
@@ -0,0 +1,48 @@
+'''
+This is our Ethernet module. It takes data from the queue that is exposed to ethernet_put,
+and whenever populated, takes that data and sends it to the host.
+'''
+
+import socket
+import logging
+from asyncio import Queue
+
+#HOST = '169.254.173.129' # IP of the host for ethernet
+HOST = '169.254.57.78'
+PORT = 65432
```

Comment:
Where do these values come from? Is there a way to get this information from some python library/hook for ip stuff, or does it have to be hardcoded in by the programmer?

Reply:
This is the Ethernet IP address of the host (the laptop) and the host port that the socket is exposed on. As far as I know this has to be hardcoded, although it would be extremely convenient if it could be automatically obtained. Maybe we could pass it in as a command line argument to make it easier to run with different laptops?

Reply:
@kimyejean02 you could make it pulled from an argparse parser object in the top level